app/assets/javascripts/device_location.js

```diff
@@ -0,0 +1,43 @@
+function showAddress(lat, lng){
+	alert(lat + ' ' + lng)
+}
```

Comment:
Are you really into this tab indentation?

With Rails it's generally the convention to use 2 spaces for everything.